lib/rentlinx/validators/phone_validator.rb

```diff
@@ -11,18 +11,12 @@ def processed_value
     private
 
     def validate
-      return if value_blank? ||
-                valid_us_number_without_country_code?(processed_value) ||
-                valid_us_number_with_country_code?(processed_value)
+      return if value_blank? || valid_phone_number?(processed_value)
       @error = "#{@value} is not a valid phone number"
     end
 
-    def valid_us_number_with_country_code?(number)
-      number.start_with?('1') && valid_us_number_without_country_code?(number[1..-1])
-    end
-
-    def valid_us_number_without_country_code?(number)
-      !number.start_with?('1') && !number.start_with?('0') && number.size == 10
+    def valid_phone_number?(number)
+      /^[1]*[2-9][0-9]{2}[2-9][0-9]{6}$/.match(number.to_s)
```

Comment:
`?` would mean 'zero or none', whereas `*` 'means zero or more times', so even if the 1 was optional we could use `?`